geoutils/georaster.py

```diff
@@ -822,10 +913,179 @@ def format_value(value):
             raise ValueError(
                 'Value provided for band was not int or None.')
 
-        if return_window == True:
+        if return_window:
             return (value, win)
         else:
             return value
-    
+
+    def xxyy(self, offset='corner', grid=True):
```

Comment:
I find the name not very explicit... maybe use self.coordinates()?

Reply:
sure

Reply:
used "coords" to keep things short